model/src/gateway/payload/incoming/auto_moderation_action_execution.rs

```diff
@@ -0,0 +1,49 @@
+use crate::{
+    guild::auto_moderation::{AutoModerationAction, AutoModerationTriggerType},
+    id::{
+        marker::{AutoModerationRuleMarker, ChannelMarker, GuildMarker, MessageMarker, UserMarker},
+        Id,
+    },
+};
+use serde::{Deserialize, Serialize};
+
+/// Message was blocked by AutoMod according to a rule.
+///
+/// Sent to bot users with [`Permissions::MANAGE_GUILD`].
```

Comment:
Applies to `rule_{create,delete,update}` too.
```suggestion
/// Only sent to bot users with [`Permissions::MANAGE_GUILD`].
// or
/// Requires [`Permissions::MANAGE_GUILD`].
```